### DISCUSSION TITLE
Is Response Streaming with IAsyncEnumerable possible via functions urls?

### DISCUSSION THREAD

**Discussion by gary-wilson** [aws/aws-lambda-dotnet#1632] [Q&A] [answered]
Function URLs support response streaming (https://docs.aws.amazon.com/lambda/latest/dg/configuration-response-streaming.html) will support for this come to dotnet?

Is there any way to currently get this working?

**Comment by plaisted** [comment]
FYI, had the same question and have a temporary workaround, see https://github.com/aws/aws-lambda-dotnet/issues/1635#issuecomment-1847502755

**Reply by ashishdhingra** [reply, marked as answer]
Feature request https://github.com/aws/aws-lambda-dotnet/issues/1635 queued in backlog.
